ui/components/MesherySettings.js

```diff
@@ -118,6 +118,7 @@ class MesherySettings extends React.Component {
       prometheus,
       tabVal,
       subTabVal,
+      isMeshConfigured: k8sconfig.clusterConfigured && grafana.grafanaURL !== '' && prometheus.prometheusURL !== '',
```

Comment:
`isMetricsConfigured` would be a better name since it is not showing the status of *mesh* but the status of metrics components

Reply:
> `isMetricsConfigured` would be a better name since it is not showing the status of _mesh_ but the status of metrics components

That makes sense. Got it, will change